app/models/employee_events.rb

```diff
@@ -0,0 +1,5 @@
+class EmployeeEvents < ApplicationRecord
+  serialize :birthday_employees, Array
+  serialize :anniversary_employees, Array
```

Comment:
These don't seem like they line up with the database schema. We have a `time_off_employees` column, but no `birthday_employees` or `anniversary_employees` columns.

Where do you expect these to be written to and read from?